ddtrace/ext/git.py

```diff
@@ -143,7 +143,7 @@ def extract_user_info(cwd=None):
 
 def extract_git_version(cwd=None):
     output = _git_subprocess_cmd("--version")
-    version_info = tuple([int(part) for part in output.split()[-1].split(".")])
+    version_info = tuple([int(part) for part in output.split()[2].split(".")])
```

Comment:
We should wrap this in a try-except and log an error if the output is in an invalid format. I am not sure if we should crash a test if we fail to extract a git version. 

Expecting `git --version` to have the following format: `git version 2.39.2` where the version only contains integers seems a bit brittle.  If the output was ``, `git version 2.39.2a1`, `git 2.49.21`, or  just `2.79.1` this function will raise a value error. I am not sure if any of these outputs are valid so feel free to ignore this suggestion.

Reply:
Good point. I will add a try except block.